README.md

```diff
@@ -6,3 +6,52 @@
 
 Packages published to [our bintray repository](https://bintray.com/apollographql/maven/federation-jvm)
 and available [in jcenter](https://jcenter.bintray.com/com/apollographql/federation/).
+
+We publish [release notes](RELEASE_NOTES.md).
+
+An example of [graphql-spring-boot](https://www.graphql-java-kickstart.com/spring-boot/) microservice is available in [spring-example](spring-example).
+
+## Getting started
+
+### Dependency management
+
+Make sure JCenter is among your repositories:
+
+```groovy
+repositories {
+    jcenter()
+}
+```
+
+Add a dependency to `graphql-java-support`:
+
+```groovy
+dependencies {
+    implementation 'com.apollographql.federation:federation-graphql-java-support:0.2.0'
+}
+```
+
+### graphql-java schema transformation
+
+`graphql-java-support` produces a `graphql.schema.GraphQLSchema` by transforming your existing schema in accordance to the
+[federation specification](https://www.apollographql.com/docs/apollo-server/federation/federation-spec/).
+It follows the `Builder` pattern.
+
+Start with `com.apollographql.federation.graphqljava.Federation.transform(…)`, which can receive either:
+- A `GraphQLSchema`;
+- A `TypeDefinitionRegistry`, optionally with a `RuntimeWiring`;
+- A String, Reader, or File declaring the schema using the Schema Definition Language, optionally with a `RuntimeWiring`;
```

Comment:
nit: link to https://www.apollographql.com/docs/apollo-server/essentials/schema/#schema-definition-language